backend/routes/recommendations.js

```diff
@@ -0,0 +1,64 @@
+const Parse = require('parse/node');
+const express = require("express")
+const router = express.Router()
+var cors = require('cors');
+
+router.use(cors())
+
+router.get('/:sessionToken', async (req, res) => {
+    //using the most recent positively reviewed book to create recs:
```

Comment:
This file will get pretty large. It might be a good idea to break it down into separate files with a single function each and then call them all in this file in order.